### DISCUSSION TITLE
how to get start first ?

### DISCUSSION THREAD

**Discussion by imycod** [lifeiteng/valle#125] [General]
I want to train a voice of my own in order to speak gpt text into speech, how do I do it? how do i do it step by step ? I have downloaded the vall-e source code

**Comment by chenjiasheng** [comment]
VALL-E is good at zero-shot cloning, it is usually used for one-sentence clones.
So it less natural/similar compared to normal TTS which trained on tens of hours of audios of target speaker. 
If you have a lot of recordings of yourself (10 hours or more), you should try the normal TTS models, such as VITS, NAR-tactron, fastspeech2.

To clone your voice with VALL-E, you don't need many of your own voice recodings.
You can follow AISHELL branch of this repo if your target language is Chinese.
